lib/mayaHydra/hydraExtensions/mayaHydraSceneProducer.cpp

```diff
@@ -50,6 +53,11 @@ MayaHydraSceneProducer::MayaHydraSceneProducer(
         initData.producer = this;
         _sceneIndex = MayaHydraSceneIndex::New(initData, lightEnabled);
         TF_VERIFY(_sceneIndex, "Maya Hydra scene index not found, check mayaHydra plugin installation.");
+        
+        //Create an HydraViewportInformation instance
+        FVP_NS_DEF::InformationInterface::ViewportInformation* hydraViewportInformation = 
+            new FVP_NS_DEF::InformationInterface::ViewportInformation(_sceneIndex, initData.cameraName, initData.viewportWidth, initData.viewportHeight, initData.rendererName);
+        FVP_NS_DEF::ViewportInformationAndSceneIndicesPerViewportDataManager::Get().AddViewportInformation(*hydraViewportInformation, _renderIndexProxy);
```

Comment:
Minor : I think these should be FVP_NS since we're in MayaHydra and thus a consumer of the Flow Viewport Toolkit? There are also a couple other instances (in mayaHydraSceneProducer.h/.cpp, and in testFlowViewportAPIViewportInformation.cpp)

Reply:
Right, I am updating them.

Reply:
Just use "Fvp".  Since we control the compilation of the Flow Viewport library, using the FVP_NS macro to account for changing the Flow Viewport library namespace name is an over elaboration, since we won't do this.

Reply:
ok I am updating it.